taxonomies/categories.txt

```diff
@@ -40566,7 +40566,7 @@ bg:Мюсли
 ca:Musli
 cs:müsli
 da:Müsli
-de:Müesli, Müslis
+de:Müsli, Müesli
```

Comment:
We kinda want plurals as the primary name, where they exist. Maybe use Müsli as an alias?

Reply:
I don't know too much about the inside of openfoodfacts, but I hoped to achieve that I can select "Müsli" (or "Müslis", I don't care) as a category. Right now, I can't do that, and I have to enter "Müesli" which is a word I've never heard before.

Reply:
Yep, me neither. Never heard of that spelling. 🤷‍♂️

Reply:
blame the swiss, that wiki link above says "...in der Schweiz und im südlichen Baden-Württemberg Müesli..." :)

Reply:
So, I changed the names to their plural forms in an additional commit. I hope, I did it right.

Reply:
looks good. thanks :)